apps/react/src/features/studios/components/StudioDetails/StudioDetails.tsx

```diff
@@ -0,0 +1,41 @@
+import { memo, FC } from 'react';
+import Typography from '@mui/material/Typography';
+import { Button } from '@mui/material';
+import { useParams } from 'react-router-dom';
+
+import styles from './StudioDetails.module.css';
+
+const StudioDetailsComponent: FC = () => {
+
+	const { studioId } = useParams<{ studioId: string; }>();
+
+	// TODO (Shmantsar A.): Implement api integration.
+	return (
+		<div>
```

Comment:
Do we actually need 'div' wrapper here. Is it possible to use [Fragment](https://react.dev/reference/react/Fragment) instead?
```suggestion
		<>
```

Reply:
![изображение](https://github.com/user-attachments/assets/08a32e1a-21f3-4bf1-8f4d-288fdf2824c9)
If I use Fragment, the page looks like this

Reply:
Ok, I get it. Thank you for clarification.
Let's use 'div' then.